terra-jupyter-aou/README.md

```diff
@@ -0,0 +1,20 @@
+# terra-jupyter-aou image
+
+This repo contains the terra-jupyter-aou image that is compatible with notebook service in [Terra]("https://app.terra.bio/") called Leonardo. For example, use `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:{version}` in terra.
+
+This image is intended to support usage in the [All of Us Workbench](https://github.com/all-of-us/workbench), which depends on Leonardo.
+
+## Image contents
+
+The terra-jupyter-aou extends the [terra-jupyter-python](../terra-jupyter-python/README.md) and [terra-jupyter-r](../terra-jupyter-r/README.md) by including the following:
+
+- Open JDK
```

Comment:
Note this image will have OpenJDK 11, but we've downgraded to OpenJDK 8 in `terra-jupyter-gatk` to support GATK.